### PR TITLE
Add possibility to queue a song

### PR DESCRIPTION
**Pull request by BeneCollyridam**
Using the new method in rspotify, we can now queue a song.
I have set the deafult to 'z', but it should probably be changed before the code
is merged.

This is in reference to issue #304 

Please let me know if there are better ways to do this, then I can improve the
code

### CODE REVIEW

**Comment by BeneCollyridam** [comment]
I am uncertain, what it should say in the help page, but maybe someone else can add that?

File: src/user_config.rs [hunk 1]
```diff
@@ -249,6 +250,7 @@ impl UserConfig {
         copy_album_url: Key::Char('C'),
         audio_analysis: Key::Char('v'),
         basic_view: Key::Char('B'),
+        add_item_to_queue: Key::Char('z'),
```
Comment (Rigellute) [inline src/user_config.rs:253]: I think you'll also want to add `add_item_to_queue` in a few other places in this file. As an example, look at the `basic_view` prop.

**Review by Rigellute — COMMENTED** [review]
Great work @BeneCollyridam - this looks great! 🎉. This has been a long requested feature.

I have only one comment on adding your new `add_item_to_queue` prop to the user config.

**Comment by Rigellute** [comment]
And you could add something like this to help.rs

```rs
 vec!["Add track to queue", "z", "Hovered over track"],
```

File: src/user_config.rs (hunk 1 above)
Comment (BeneCollyridam) [reply src/user_config.rs:254]: I believe I have fix this in eb6416d

**Comment by BeneCollyridam** [comment]
I added your (@Rigellute) recommended help text. Thanks!

**Review by Rigellute — APPROVED** [review]
Awesome 🎉!

**Comment by Rigellute** [comment]
@all-contributors please add @BeneCollyridam for code

**Pull request merged**